api/jest.utils.ts

```diff
@@ -66,7 +66,8 @@ export function setupServer(): void {
     await fastify.ready();
 
     global.fastifyTestInstance = fastify;
-  });
+    // allow a little time to setup the db
+  }, 10000);
```

Comment:
In my PR, this would add 120s to the test time?

Reply:
It's an upper limit and the push is once per call to `setupServer`, which we do once per suite.

When I run it locally it takes the entire suite from about 55 seconds to 75.

Reply:
Ah. Thank you. I looked at the docs for `beforeAll` and interpreted it as a `setTimeout`.

Reply:
Yeah, it's just giving it more time for the extra setup.